server/src/main/java/io/crate/types/TimestampType.java

```diff
@@ -187,12 +195,12 @@ static long parseTimestampIgnoreTimeZone(String timestamp) {
         .parseCaseInsensitive()
         .append(ISO_LOCAL_DATE)
         .optionalStart()
-            .padNext(1)
-                .optionalStart()
-                    .appendLiteral('T')
-                .optionalEnd()
-            .append(ISO_LOCAL_TIME)
-            .optionalStart()
```

Comment:
Could we keep the indentation? The additional structure reflects the `start` / `end` markers